official/modeling/fast_training/experimental/tf2_utils_2x_wide.py

```diff
@@ -16,7 +16,7 @@
 
 from absl import logging
 import numpy as np
-import tensorflow as tf, tf_keras
+import tensorflow as tf
```

Comment:
Please revert the import of `tf_keras` to be consistent with other files.

Reply:
Fixed import.